### ISSUE TITLE
Show History of Lookup Table Changes

### ISSUE THREAD

**Issue by sean-usds** [CDCgov/prime-reportstream#3792]
## Problem Statement
Admins (Rick Hood to start), need a web UI to see the changes that have been made to a lookup table.

## Acceptance Criteria
- Can see the history of changes to a lookup table
- Can download the table values from a specific time in the history

## Assumptions
- API endpoints do **NOT** already exists

## Blocked By
- https://app.zenhub.com/workspaces/experience-607d9d5e68b95200150fec37/issues/cdcgov/prime-reportstream/3790

**Comment by HeatherRienks** [comment]
Need to split into 2 tickets - Eng & Design
 @bgantick to create Eng

**Comment by HeatherRienks** [comment]
Need 2 separate tickets, one for Engineering and one for Design.

@bgantick to create a new Engineering ticket.

**Comment by bgantick** [comment]
Going to open a new ticket for this
